lib/generate/collector/category.js

```diff
@@ -31,6 +32,8 @@ const category = async (
 
     const page = await browserContext.newPage();
 
+    timeout && await page.setDefaultNavigationTimeout(timeout);
```

Comment:
Can you please change this to be a standard `if (timeout) {` statement here and in all the other files? Additional bytes don't matter in our case and I find it more readable for everyone.